README.md

```diff
@@ -0,0 +1,88 @@
+# README
+
+## Introduction
+
+This script is an example implementation of a shielded transfer using the Light Protocol and **Solana** blockchain. It demonstrates how to initialize a **Solana** wallet, request an airdrop of SOL tokens, set up a test relayer, perform a shield operation, transfer tokens to a recipient, and retrieve transaction information.
+
+## Prerequisites
+
+Before running this script, ensure that you have the following prerequisites:
+
+1. **Node**.js and **npm** installed on your machine.
+2. Access to a **Solana** node running locally.
```

Comment:
- actually not local-test starts a test-validator for you
- also if you just have a normal test validator running you will need to initialize the merkle tree and other accounts manually or using the setMerkleTree script

Reply:
✅